package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "boilerplate-web-app",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
The API change is deemed non-breaking - `POST /api/v0/characters` went from not returning a response body to returning a response body, so a client would only break if it was actively insisting on an empty response body.